ui/src/components/Map/utils.ts

```diff
@@ -18,4 +18,16 @@ const numericSeverityLevel = (severity: string | undefined) => {
   return 0
 }
 
-export { numericSeverityLevel }
+const toFixed = (s: string, n: number) => {
```

Comment:
Does this function need a better name, or maybe a description of what it's for otherwise? `stringToFixedFloat` maybe? The arguments can also help out here with the self-documentation, maybe s => floatAsString and n => decimalPoints or fixedLength.

Reply:
Sounds good, will do in upcoming PR.